Project.toml

```diff
@@ -11,19 +11,22 @@ Tricks = "410a4b4d-49e4-4fbc-ab6d-cb71b17b3775"
 
 [weakdeps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 Measurements = "eff96d63-e80a-5855-80a2-b1b0885c5ab7"
 ScientificTypes = "321657f4-b219-11e9-178b-2701a2544e81"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [extensions]
 DynamicQuantitiesLinearAlgebraExt = "LinearAlgebra"
+DynamicQuantitiesMakieExt = "Makie"
 DynamicQuantitiesMeasurementsExt = "Measurements"
 DynamicQuantitiesScientificTypesExt = "ScientificTypes"
 DynamicQuantitiesUnitfulExt = "Unitful"
 
 [compat]
 DispatchDoctor = "0.4"
 LinearAlgebra = "1"
+Makie = "0.22.2"
```

Comment:
Can you comment on why choose this version as a minimum compat?

Reply:
I think I was looking at Measurements.jl for prior art in adding Makie as a package extension when I was just getting started, and was just being conservative. Looking back, I think this could be [relaxed to Makie v0.21.0](https://github.com/MakieOrg/Makie.jl/blob/b9b5ba3ca1292cef03f1edaee5bb35b376dd954c/CHANGELOG.md?plain=1#L316) since it looks like that is when https://github.com/MakieOrg/Makie.jl/pull/3226 was added

Reply:
1277f6f